autogen/docs/behaviorspace.md.mustache

```diff
@@ -212,6 +213,23 @@ the run in some other way.
 don't want to set any maximum, but want the length of the runs to be controlled
 by the stop condition instead, enter 0.
 
+### Importing and exporting
+
+Although experiments are tied to a model and are usually saved along with a
+model, they can also be imported and exported individually. This allows you to
+easily transfer experiments between models, and also prepares experiments to be
+run headlessly.
```

Comment:
Maybe point out the experiment XML can be hand-edited in a text editor or edited in another script/language for running headlessly as another benefit here.